source/jsonnet/northern-ireland/household/blocks/who-lives-here/do_you_usually_live_here.jsonnet

```diff
@@ -53,7 +53,7 @@ local questionTitle = {
     },
   },
   question: {
-    id: 'primary-confirmation-question',
+    id: 'do-you-usually-live-here-question',
```

Comment:
The answer ids could be made consistent with this.

Reply:
Changed so it's similar to ENG schemas file.👍